misk-config/src/main/kotlin/misk/config/MiskConfig.kt

```diff
@@ -155,7 +155,7 @@ object MiskConfig {
       }
 
       val path = Joiner.on('.').join(e.path.map { it.fieldName ?: it.index })
-      logger.warn(e) {
+      logger.warn {
```

Comment:
On the flip side the full stack trace probably shows where the call site is trying to access the non-existent field?